api/cluster-resources/py-dataset-snippets-menu.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "All of Us Dataset Builder Python snippets",
   "sub-menu": [
-    {
-      "name": "Setup",
-      "snippet": "!pip3 install --upgrade --user pandas_profiling statsmodels\n\nimport os\nimport numpy as np\nimport pandas as pd\nimport pandas_profiling\nimport plotnine\nfrom plotnine import *  # Provides a ggplot-like interface to matplotlib.\n\n## Plot setup.\ntheme_set(theme_bw(base_size = 11)) # Default theme for plots.\n\ndef get_boxplot_fun_data(df):\n  \"\"\"Returns a data frame with a y position and a label, for use annotating ggplot boxplots.\n\n  Args:\n    d: A data frame.\n  Returns:\n    A data frame with column y as max and column label as length.\n  \"\"\"\n  d = {'y': max(df), 'label': f'N = {len(df)}'}\n  return(pd.DataFrame(data=d, index=[0]))\n\n# NOTE: if you get any errors from this cell, restart your kernel and run it again.\n"
-    },
     {
       "name": "Documentation",
       "external-link": "https://github.com/all-of-us/workbench-snippets"
     },
     "---",
     {
-      "name": "Basic operations",
+      "name": "(1) Setup",
```

Comment:
nit: Is there a way to avoid encoding the ordinals directly in the name field and having those applied at render time? That would make it easier to reorder things.